auth/src/api/handlers.rs

```diff
@@ -68,6 +65,20 @@ pub(crate) async fn update_user_tier(
     Ok(())
 }
 
+// #[debug_handler]
```

Comment:
Not sure what this is? :sweat_smile:

Reply:
Remnants of past trauma. :joy: Will remove.